### PR TITLE
Prepare Cortex 1.8.0 (final) release.

### PR DESCRIPTION
**Pull request by pstibrany**
**What this PR does**: Prepare Cortex 1.8.0 release.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -98,11 +98,11 @@ To publish a release candidate:
...
 1. Do not change the release branch directly; make a PR to the release-X.Y branch with VERSION and any CHANGELOG changes.
-1. Ensure the `VERSION` file has **no** `-rc.X` suffix
-1. Update the Cortex version in the following locations:
-   - Kubernetes manifests located at `k8s/`
-   - Documentation located at `docs/`
-1. `git tag` the new release (see [How to tag a release](#how-to-tag-a-release))
+   1. Ensure the `VERSION` file has **no** `-rc.X` suffix
+   1. Update the Cortex version in the following locations:
+      - Kubernetes manifests located at `k8s/`
+      - Documentation located at `docs/`
```
Comment (gouthamve) [inline RELEASE.md:104]: Was this intentional?

**Review by gouthamve — APPROVED** [review]
LGTM!

File: RELEASE.md (hunk 1 above)
Comment (pstibrany) [reply RELEASE.md:104]: Yes. I wanted to express that these points are part of first (make PR to the release branch...). PR should include these changes.

**Review by pracucci — APPROVED** [review]
LGTM, thanks for working on the release! 🙏

**Pull request merged**